### ISSUE TITLE
Error: Cannot open database: file missing - Is another copy of Syncthing already running?

### ISSUE THREAD

**Issue by HeMaN-NL** [bloones/SyncThingWin#1]
I tried to install the syncthing service via syncthingtray but get the error below when the service is started.
After this error the service is stopped and syncthing as well.

System is Win7 Ultimate x64
Program files in: C:\Program Files\Syncthing\

Original config and all other files are copied from the original location to C:\Program Files\Syncthing\config (which I created first)

Any ideas what I can look into?

Log:

```
[UC64X] 17:21:58 INFO: syncthing v0.10.13 (go1.4 windows-amd64 default) unknown-user@syncthing-builder 2014-12-18 21:32:03 UTC
[UC64X] 17:21:58 INFO: My ID: xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx
[UC64X] 17:21:58 FATAL: Cannot open database: file missing - Is another copy of Syncthing already running?
[UC64X] 17:22:00 INFO: syncthing v0.10.13 (go1.4 windows-amd64 default) unknown-user@syncthing-builder 2014-12-18 21:32:03 UTC
[UC64X] 17:22:00 INFO: My ID: xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx
[UC64X] 17:22:00 FATAL: Cannot open database: file missing - Is another copy of Syncthing already running?
[UC64X] 17:22:01 INFO: syncthing v0.10.13 (go1.4 windows-amd64 default) unknown-user@syncthing-builder 2014-12-18 21:32:03 UTC
[UC64X] 17:22:01 INFO: My ID: xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx
[UC64X] 17:22:01 FATAL: Cannot open database: file missing - Is another copy of Syncthing already running?
[UC64X] 17:22:03 INFO: syncthing v0.10.13 (go1.4 windows-amd64 default) unknown-user@syncthing-builder 2014-12-18 21:32:03 UTC
[UC64X] 17:22:03 INFO: My ID: xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx-xxxxxxx
[UC64X] 17:22:03 FATAL: Cannot open database: file missing - Is another copy of Syncthing already running?

```

**Comment by bloones** [comment]
Can you check the following:

Is syncthing already running? Try to open the Syncthing Web GUI and if you can, then shutdown syncthing from there and start the service again.

If syncthing is not running, execute syncthingtray with the configure option (syncthingtray configure) and check that the path indicated in Configuration does contain a config.xml file.

The most likely reason, is that syncthing is already running: I had the same issue once.

**Comment by HeMaN-NL** [comment]
checked;
Syncthing not running (no webgui available (404), and did a reboot of the system just to be sure of no other processes.
The path indicatec does contain the config xml, when setting up syncthingtray it copied all files from the original location, including the config.xml 

Uninstalled the service, rebooted, checked existence of a syncthing proces (not there), installed syncthingtray service again, still the same error :(

**Comment by HeMaN-NL** [comment]
Think I found the problem, my database was corrupted I suppose. 
First time I installed the service I think Syncthing was still running, and think there was a warning about a file being locked (not sure anymore).

After removing the contents of the index folder in the configuration folder was able to start syncthing tray, and it started rebuilding the index, and was able to create secure connections to my other nodes.

thnx!! Will have a closer look tomorrow :D
